system/kube-monitoring/charts/prometheus-collector/prometheus.yaml

```diff
@@ -232,6 +232,11 @@ scrape_configs:
     target_label: __address__
     regex: ([^:]+)(:\d+)?
     replacement: ${1}:9100
+  - source_labels: [mountpoint]
```

Comment:
This might work. Not 100% sure. You either need this rule or the aggregated rules.